### DISCUSSION TITLE
Domains issue

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1531] [Q&A]
Hello!
When I open the domain simplelogin.com, I get redirected to simplelogin.io (this is correct), but when I open the other domains (aleeas.com, dralias.com, 8shield.net), there is no redirect.

However, when opening the domain slmail.me, there is a redirect to simplelogin.io.

Solution to the problem:
In the case of opening domains aleeas.com, dralias.com, 8shield.net through the browser make a redirect to simplelogin.io

**Comment by nguyenkims** [comment]
Thanks for reporting, this should be fixed now.
